src/libs/ck-libs/ampi/ampiCC

```diff
@@ -75,6 +80,14 @@ done
 
 eval processArgs "$@"
 
+# If there are no args except -v or --version specified, just run
+# charmc with -v or --version. Helps with configure scripts that
+# parse the output to detect compiler versions.
+if [[ -z $ARGS ]] && [[ -n $SHOW_COMPILER_VERSION ]]; then
```

Comment:
Should `$ARGS` and `$SHOW_COMPILER_VERSION` be quoted here and once below or is the quoting practice for bash's `[[` different from `[` and `test`?

Reply:
Yeah, with `[[` you don't need to quote variables anymore (see e.g. https://stackoverflow.com/questions/3869072/test-for-non-zero-length-string-in-bash-n-var-or-var). If you prefer, I can add quotes if it makes it more consistent with the rest of the script.

Reply:
Added quotes.